org.civicrm.reqangular/test/decorators/ui-bootstrap/uib-modal.decorator.spec.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-env amd, jasmine */
+
+define([
+  'common/lodash',
+  'common/angularMocks',
+  'common/angular',
+  'common/decorators/ui-bootstrap/uib-modal.decorator',
+  'common/angularBootstrap'
+], function (_, angularMocks, angular, $uibModalDecorator, $document) {
+  'use strict';
+
+  describe('$uibModal.open', function () {
+    var $uibModal, $provide, $document;
+
+    beforeEach(module('ui.bootstrap'));
+
+    beforeEach(function () {
+      module(function (_$provide_) {
+        $provide = _$provide_;
+      });
+      inject(function () {
+        $provide.decorator('$uibModal', $uibModalDecorator);
+      });
+    });
+
+    beforeEach(inject(function (_$uibModal_, _$document_) {
+      $uibModal = _$uibModal_;
+      $document = _$document_;
+    }));
+
+    describe('init', function () {
+      var arg1 = { template: '<component></component>' };
+      var arg2 = 'another_argument';
+      var elements = {};
+      var doc, originalStyles;
+
+      beforeEach(function () {
+        doc = $document[0];
```

Comment:
1) if you are using `doc` only in the `beforeEach` callback, then you don't need it declared it in the outer function
2) I don't think this variable is needed at all, just use `$document[0]` directly